### PR TITLE
New tests

### PR DESCRIPTION
**Pull request by mabar**
Related to #26 (but not closing yet)

### CODE REVIEW

File: src/Schema/Validation/ControllerPathValidation.php [hunk 1]
```diff
@@ -22,12 +22,9 @@ protected function validateSlashes(SchemaBuilder $builder): void
 		foreach ($controllers as $controller) {
 			$path = $controller->getPath();
 
-			if (strlen($path) === 1) {
-				if ($path === '/') continue;
-
-				// MUST: Be exactly /
+			if ($path === null) {
```
Comment (f3l1x) [inline src/Schema/Validation/ControllerPathValidation.php]: Why have you changed it?

Comment (mabar) [reply src/Schema/Validation/ControllerPathValidation.php]: Exception was misleading. @ControllerPath does not must be `/`, it just could be only `/`. Check if path starts with slash is enough for that.

Comment (f3l1x) [reply src/Schema/Validation/ControllerPathValidation.php]: Ouk

**Comment by f3l1x** [comment]
I've merged it. Keep working. ;-)

**Pull request merged**